src/openrct2/platform/Platform.Android.cpp

```diff
@@ -70,6 +70,11 @@ namespace Platform
     {
         return MeasurementFormat::Metric;
     }
+
+    std::string GetSteamPath()
+    {
+        return "";
```

Comment:
I'd return `std::string()` or `{}`, there's not much of a difference, but I think it is clearer (?).

This would apply to multiple places of the patch.

Reply:
I agree, best to return a default constructed std::string via ```{}```. This is also beneficial for debug builds as with no optimizations it would still try to copy that empty thing.

Reply:
Also, returning `{}` prevents you from having to change more of the functions if the type ever changes